gradle/quality.gradle

```diff
@@ -1,7 +1,6 @@
 apply plugin: 'findbugs'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
-apply plugin: 'jacoco'
```

Comment:
afaik this was for code coverage numbers, any reason we are removing it?

Reply:
Its not removed, this was redundant code : https://github.com/mesosphere/dcos-commons/blob/master/build.gradle#L21